### PR TITLE
Add waiters and example for instance operations in `sqlserverflex` module

### PR DESCRIPTION
**Pull request by joaopalet**

### CODE REVIEW

File: services/mongodbflex/wait/wait_test.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func TestCreateInstanceWaitHandler(t *testing.T) {
 
 			handler := CreateInstanceWaitHandler(context.Background(), apiClient, "", instanceId)
 
-			gotRes, err := handler.SetTimeout(10 * time.Millisecond).WaitWithContext(context.Background())
+			gotRes, err := handler.SetTimeout(10 * time.Millisecond).SetSleepBeforeWait(1 * time.Millisecond).WaitWithContext(context.Background())
```
Comment (DiogoFerrao) [inline services/mongodbflex/wait/wait_test.go:129]: Great catch 👍

File: examples/sqlserverflex/sqlserverflex.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+	"github.com/stackitcloud/stackit-sdk-go/core/config"
+	"github.com/stackitcloud/stackit-sdk-go/core/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex/wait"
+)
+
+func main() {
+	// Specify the project ID
+	projectId := "16f49d71-37ad-4137-8b97-44d9c55c4094"
```
Comment (DiogoFerrao) [inline examples/sqlserverflex/sqlserverflex.go:16]: This should be a placeholder

File: examples/sqlserverflex/sqlserverflex.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+	// Specify instance configuration options
+	// flavorId := "FLAVOR_ID"
+	// version := "VERSION"
```
Comment (DiogoFerrao) [inline examples/sqlserverflex/sqlserverflex.go:20]: I think we can uncomment these and delete the actual flavor and version below. This way we dont induce errors if the flavors/versions become deprecated.

File: examples/sqlserverflex/sqlserverflex.go (hunk 2 above)
Comment (joaopalet) [reply examples/sqlserverflex/sqlserverflex.go]: Ups, these were leftovers from testing. Didn't mean to commit them

File: examples/sqlserverflex/sqlserverflex.go (hunk 3 above)
Comment (joaopalet) [reply examples/sqlserverflex/sqlserverflex.go]: Same as above

**Review by DiogoFerrao — APPROVED** [review]

**Pull request merged**